metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/DepublishPlugin.java

```diff
@@ -63,6 +63,8 @@ public DpsTask prepareDpsTask(String datasetId,
         extraParameters.put(PluginParameterKeys.RECORD_IDS_TO_DEPUBLISH, recordIdList);
       }
     }
+    //TODO: 2024-09-24 - Update below key with the PluginParameterKeys equivalent when it's available
```

Comment:
is there a ticket for this part?

Reply:
Ok I just created 6182 part of the depublication topology preparations(6147) and also 6183 to cleanup this and the deprecations mentioned earlier this morning